frontend/src/components/StudentProfile/AuditHistory/CourseChangeHistory.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <v-data-table
+      :sortBy="sortBy"
+      :items="courseHistory"
+      :headers="courseFields"
+      :items-per-page="'-1'"
+      item-value="id"
+  >
+    <template v-slot:headers="{ columns, isSorted, getSortIcon, toggleSort }">
+      <tr>
+        <template v-for="column in columns" :key="column.key">
+          <th>
+            <div class="d-flex align-center">
+              <span
+                  class="me-2 cursor-pointer"
+                  @click="toggleSort(column)"
+                  v-text="column.title"
+                  :title="column.description"
+              ></span>
+              <v-icon
+                  v-if="isSorted(column)"
+                  :icon="getSortIcon(column)"
+                  color="medium-emphasis"
+              ></v-icon>
+            </div>
+          </th>
+        </template>
+      </tr>
+    </template>
+    <template v-slot:expanded-row="{ item }">
+      <tr>
+        <td :colspan="courseFields.length">
+          <v-row dense>
+            <v-col
+                v-for="field in courseExamFields"
+                :key="field.key"
+                class="text-center"
+                :title="field.description"
+            >
+              <div class="text-caption font-weight-bold">{{ field.label }}</div>
+              <div class="text-body-2">{{ item.courseExam[field.key] ?? '' }}</div>
+            </v-col>
+          </v-row>
+        </td>
+      </tr>
+    </template>
+    <template
+        v-slot:item.data-table-expand="{
+                item,
+                internalItem,
+                toggleExpand,
+                isExpanded,
+              }"
+    >
+      <td v-if="!!item.courseExam">
+        <v-btn
+            variant="text"
+            density="comfortable"
+            size="small"
+            @click="toggleExpand(internalItem)"
+            class="v-data-table__expand-icon"
+            :class="{ 'v-data-table__expand-icon--active': isExpanded }"
+            :icon="
+                    isExpanded(internalItem)
+                      ? 'mdi-chevron-down'
+                      : 'mdi-chevron-right'
+                  "
+        >
+        </v-btn>
+      </td>
+    </template>
+    <template v-slot:item.activityCode="{ item }">
+      <i>{{ item.activityCode }}</i> -
+      {{ item.activityDescription }}
+    </template>
+    <template v-slot:item.updateDate="{ item }">
+      {{ $filters.formatTime(item.updateDate) }}
+    </template>
+  </v-data-table>
+</template>
+
+<script>
+
+import {mapActions, mapState} from "pinia";
+import {useStudentStore} from "@/store/modules/student";
+
+export default {
+  props: {},
+  computed: {
+    ...mapState(useStudentStore, {
+      studentId: "getStudentId",
+      courseHistory: "getStudentCourseAuditHistory"
+    }),
+  },
+  data: function () {
+    return {
+      courseChangeHistory: [],
+      courseFields: [
```

Comment:
Rename this to courseHeaders.

Bootstrap used fields but vuetify uses headers, so some code still says fields but we're working on changing that as we make changes to each component.